CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_zendesk v0.19.1
+[PR #180](https://github.com/fivetran/dbt_zendesk/pull/180) includes the following update:
+
+## Bug Fixes
+- In v0.19.0, Snowflake users may have seen `when searching for a relation, dbt found an approximate match` errors when running the `stg_zendesk__group_tmp` model. We have updated the `adapter.get_relation()` [logic](https://github.com/fivetran/dbt_zendesk_source/blob/main/macros/union/union_zendesk_connections.sql) that was causing this error in the `zendesk_source` [package](https://github.com/fivetran/dbt_zendesk_source).
```

Comment:
Suggestion to link to the source release as well.

Reply:
added